src/etc/test-float-parse/Cargo.toml

```diff
@@ -8,7 +8,24 @@ publish = false
 resolver = "1"
 
 [dependencies]
-rand = "0.8"
+indicatif = { version = "0.17.8", default-features = false }
+num = "0.4.3"
+rand = "0.8.5"
+rand_chacha = "0.3"
+rayon = "1"
 
 [lib]
 name = "test_float_parse"
+
+# Bigint libraries are slow without optimization, speed up testing
+[profile.dev.package."*"]
+opt-level = 3
+
+# Squeeze out the last bit of available speed
+[profile.release]
+codegen-units = 1
+lto = true
+
+[profile.profiling]
+inherits = "release"
+debug = true
```

Comment:
I guess the thinking is that the super long runtime on these tests makes this merited above the normal status quo? I'd rather see these move to bootstrap (e.g., by setting Cargo env variables for configuring these properties) or into the root Cargo.toml per a previous comment, both of which I believe would retain the customization that does seem justified for multi-hour test runs.

If those options are too annoying/difficult, then let's forgo moving this crate into the workspace and instead make sure it's part of the tidy dependency checking.

Reply:
Ah I only had this for when I was trying to debug a slow test. I’ll remove it.